[Bug]: test_health_score_changes_based_on_alert_severity failing due to mock alert not triggered

### Description
https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/all/51249/2220535/2220910/log

/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/functional/ui/test_health_overview.py:566: 

/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/functional/ui/test_health_overview.py:192: in wait_for_health_score_change
    for score in TimeoutSampler(timeout, 60, self.get_health_score_ui):
  raise TimeoutExpiredError(self._get_exception_log(exp=last_exp), last_exp)
E       timeout_sampler.TimeoutExpiredError: Timed Out: 300
E       Function: tests.functional.ui.test_health_overview.get_health_score_ui
E       Last exception: N/A: None


### Actual Behavior
Mock alert ODFNodeNICBandwidthSaturation is not triggered

### Expected Behavior
Mock alert ODFNodeNICBandwidthSaturation should be triggered

### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)

### Screenshots (if applicable)

### Environment
- Test Suite(s):
- Platform(s):
- Version(s):
- OS:

### Additional Context

--- a/ocs_ci/templates/health_overview_alerts/custom-odf-latency-rule.yaml
+++ b/ocs_ci/templates/health_overview_alerts/custom-odf-latency-rule.yaml
@@ -14,7 +14,7 @@ spec:
       annotations:
         description: Node {{ $labels.instance }} has max ICMP RTT latency > 5ms over the last 1min. This may impact Ceph performance.
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/ODFNodeLatencyHighOnOSDNodes.md
-        summary: High ICMP latency (>1ms) detected on ODF OSD node(s)
+        summary: High ICMP latency (>5ms) detected on ODF OSD node(s)
       expr: |
         count(
              max_over_time(

--- a/ocs_ci/templates/health_overview_alerts/custom-odf-latency-rule.yaml
+++ b/ocs_ci/templates/health_overview_alerts/custom-odf-latency-rule.yaml
@@ -12,7 +12,7 @@ spec:
     rules:
     - alert: ODFNodeLatencyHighOnOSDNodes   # SAME NAME
       annotations:
-        description: Node {{ $labels.instance }} has max ICMP RTT latency > 1ms over the last 1min. This may impact Ceph performance.
+        description: Node {{ $labels.instance }} has max ICMP RTT latency > 5ms over the last 1min. This may impact Ceph performance.
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/ODFNodeLatencyHighOnOSDNodes.md
         summary: High ICMP latency (>1ms) detected on ODF OSD node(s)
       expr: |

--- a/ocs_ci/templates/health_overview_alerts/custom-odf-latency-rule.yaml
+++ b/ocs_ci/templates/health_overview_alerts/custom-odf-latency-rule.yaml
@@ -16,12 +16,19 @@ spec:
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/ODFNodeLatencyHighOnOSDNodes.md
         summary: High ICMP latency (>1ms) detected on ODF OSD node(s)
       expr: |
-        max_over_time(
-        probe_icmp_duration_seconds{
-          phase="rtt",
-          job="odf-blackbox-exporter",
-          daemon_type="osd"}[24h: ]
-        ) > 0.005
+        count(
+             max_over_time(
+               probe_icmp_duration_seconds{phase="rtt",
+               job=~"probe/.*/odf-blackbox-exporter",
+               daemon_type="osd"
+               }[24h:]
+             ) > 0.005
+             and on(instance, job, daemon_type, namespace)
+             probe_success{
+               job=~"probe/.*/odf-blackbox-exporter",
+               daemon_type="osd"
+             } == 1
+            ) > 0
       for: 1m
       labels:
         component: odf

--- a/ocs_ci/templates/health_overview_alerts/custom-odf-node-latency-high-non-osd.yaml
+++ b/ocs_ci/templates/health_overview_alerts/custom-odf-node-latency-high-non-osd.yaml
@@ -9,13 +9,19 @@ spec:
       rules:
         - alert: ODFNodeLatencyHighOnNonOSDNodes
           expr: |
-            max_over_time(
-              probe_icmp_duration_seconds{
-                phase="rtt",
-                job="odf-blackbox-exporter",
-                daemon_type="mon"
-              }[24h:]
-            ) > 0.005
+            count(
+             max_over_time(
+               probe_icmp_duration_seconds{phase="rtt",
+               job=~"probe/.*/odf-blackbox-exporter",
+               daemon_type="mon"
+               }[24h:]
+             ) > 0.005
+             and on(instance, job, daemon_type, namespace)
+             probe_success{
+               job=~"probe/.*/odf-blackbox-exporter",
+               daemon_type="mon"
+             } == 1
+            ) > 0
           for: 1m
           labels:
             component: odf

--- a/tests/functional/ui/test_health_overview.py
+++ b/tests/functional/ui/test_health_overview.py
@@ -615,9 +615,7 @@ class TestHealthOverview(ManageTest):
             api = PrometheusAPI(threading_lock=threading_lock)
             api.refresh_connection()
             alerts = api.wait_for_alert(name=alert_name, state="firing", sleep=60)
-            assert (
-                len(alerts) > 0
-            ), f"Alert {alert_name} is not in firing state after unsilencing"
+            assert len(alerts) > 0, f"Alert {alert_name} is not in firing state"
             logger.info(f"Alert {alert_name} confirmed in firing state")
         self.update_alert_map(threading_lock)
         logger.info("Silence all pre-existing alerts")

--- a/tests/functional/ui/test_health_overview.py
+++ b/tests/functional/ui/test_health_overview.py
@@ -612,6 +612,13 @@ class TestHealthOverview(ManageTest):
             self.restart_pod()
             logger.info("Waiting for 120 sec for pod to restart")
             time.sleep(120)
+            api = PrometheusAPI(threading_lock=threading_lock)
+            api.refresh_connection()
+            alerts = api.wait_for_alert(name=alert_name, state="firing", sleep=60)
+            assert (
+                len(alerts) > 0
+            ), f"Alert {alert_name} is not in firing state after unsilencing"
+            logger.info(f"Alert {alert_name} confirmed in firing state")
         self.update_alert_map(threading_lock)
         logger.info("Silence all pre-existing alerts")
         df_overview = PageNavigator().nav_storage_data_foundation_overview_page()
@@ -657,7 +664,9 @@ class TestHealthOverview(ManageTest):
             api = PrometheusAPI(threading_lock=threading_lock)
             api.refresh_connection()
             alerts = api.wait_for_alert(name=alert_name, state="firing", sleep=60)
-            logger.info(f"Alert {alerts} triggered")
+            assert (
+                len(alerts) > 0
+            ), f"Alert {alert_name} was not triggered after applying mock alert rule"
             PageNavigator().take_screenshot(f"triggered_alert_{alert_name}")
             logger.info(
                 "Waiting for 120 sec to update health score after alert is triggered"

